style.css

```diff
@@ -179,57 +179,45 @@ body {
     font-weight: 700;
 }
 
-#lvl-text, #current-lvl {
-    color: rgb(213, 216, 38);
+
+#ui-overlay-top-left {
+    position: absolute;
+    z-index: 99;
 }
 
-#player-top-left {
+#lvl-container {
     position: absolute;
-    left: 0;
-    top: 0;
-    background-color: black;
-    border-radius: 4px;
-    width: 20vh;
-    height: 20vh;
```

Comment:
**Avoid** `vh` & `vw` at all costs from now, since we now have a restricted dimension in `pixels` (of up to 1920x1080)

So either use `px` (with media queries) or `%` (for smooth scaling) when you're working with stuff